rails/rubocop.yml

```diff
@@ -1,3 +1,11 @@
 Rails/SafeNavigation:
   Description: Converts usages of try! or try to &..
   ConvertTry: true
+
+Rails/UnknownEnv:
+  Environments:
+    - development
+    - test
+    - production
+    - staging
+    - heroku
```

Comment:
do we need `heroku` here? we dont ever use it explicitly - it's being used in environments folder as a base for staging / production / demo
also, might be good to add `demo`
```
nucleus ⭠ root ❯ heroku config -a thanx-demo | ag RAILS
RAILS_ENV:                         demo
```

Reply:
👍